resources/Arduino/devkit/template.json

```diff
@@ -12,6 +12,19 @@
                     "targetPath" : "./device"
                 }
             ]
+        },
+        {
+            "description": "",
+            "detail": "Create project that connects to Azure IoT Hub and processes device data further in Azure Functions.",
+            "label": "With Azure Functions",
+            "type": "AzureFunctions",
+            "templateFilesInfo": [
+                {
+                    "fileName": "simple_iothub.ino",
+                    "sourcePath" : ".",
+                    "targetPath" : "./device"
+                }
+            ]
```

Comment:
For the support of Azure Functions, there are some additional tool to be installed on target machine.
https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions
Does your docker file contain all those information?

Reply:
The current version of iot workbench will suggest install the azure-function extension when user choose "With Azure Functions" devkit project. This method encounters one issue that currently vscode-insider does not provide accessible link like` vscode:extension/ms-azuretools.vscode-azurefunctions`.

As you suggested, we can directly choose a docker image with azure function extension pre-installed for users if they choose to create a "With Azure Functions" devkit project.
I will fix the issue next working day.

Reply:
Fixed.